src/mscorlib/src/System/CurrentSystemTimeZone.cs

```diff
@@ -29,12 +29,8 @@ namespace System
 {
     [Obsolete("System.CurrentSystemTimeZone has been deprecated.  Please investigate the use of System.TimeZoneInfo.Local instead.")]
     [Serializable]
-    internal class CurrentSystemTimeZone : TimeZone
+    internal partial class CurrentSystemTimeZone : TimeZone
     {
-        // The per-year information is cached in in this instance value. As a result it can
-        // be cleaned up by CultureInfo.ClearCachedData, which will clear the instance of this object
```

Comment:
There is a bug: CultureInfo.ClearCachedData() does not clear this table. The code was under `FEATURE_CORECLR` ifdef originally, but it was brought back together with CurrentSystemTimeZone. Could you please fix it while you are on it?